lib/fathead/mdn_css/output.txt

```diff
@@ -523,7 +523,7 @@ columns	A			css properties							<section class="prog__container"><p>The columns
 columns property	R	columns										
 comments	A										<section class="prog__container"><p>Comments are used to add explanatory notes or prevent the browser from interpreting parts of the stylesheet.</p><pre><code>/* Comment */</code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/Comments
 computed value	A										<section class="prog__container"><p>The computed value of a CSS property is computed from the specified value by:</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/computed_value
-content	A			css properties							<section class="prog__container"><p>The content CSS property is used with the ::before and ::after pseudo-elements to generate content in an element. Objects inserted using the content property are anonymous replaced elements.</p><p><b>Initial Value: </b><em>normal</em></p><pre><code>/* Keywords that cannot be combined with other values */\ncontent: normal;\ncontent: none;\n\n/* <string> value, non-latin characters must be encoded */\n/* e.g. \00A0 for � */\ncontent: &#39;prefix&#39;;\n\n/* <uri> value */\ncontent: url(http://www.example.com/test.png);\n\n/* <counter> values */\ncontent: chapter_counter;\n\n/* attr() value linked to the HTML attribute value */\ncontent: attr(value string);\n\n/* Language- and position-dependant keywords */\ncontent: open-quote;\ncontent: close-quote;\ncontent: no-open-quote;\ncontent: no-close-quote;\n\n/* Except for normal and none, several values */\n/* can be used simultaneously */\ncontent: open-quote chapter_counter;\n\n/* Global values */\ncontent: inherit;\ncontent: initial;\ncontent: unset;\n</counter></uri></string></code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/content
+content	A			css properties							<section class="prog__container"><p>The content CSS property is used with the ::before and ::after pseudo-elements to generate content in an element. Objects inserted using the content property are anonymous replaced elements.</p><p><b>Initial Value: </b><em>normal</em></p><pre><code>/* Keywords that cannot be combined with other values */\ncontent: normal;\ncontent: none;\n\n/* <string> value, non-latin characters must be encoded */\n/* e.g. \\00A0 for � */\ncontent: &#39;prefix&#39;;\n\n/* <uri> value */\ncontent: url(http://www.example.com/test.png);\n\n/* <counter> values */\ncontent: chapter_counter;\n\n/* attr() value linked to the HTML attribute value */\ncontent: attr(value string);\n\n/* Language- and position-dependant keywords */\ncontent: open-quote;\ncontent: close-quote;\ncontent: no-open-quote;\ncontent: no-close-quote;\n\n/* Except for normal and none, several values */\n/* can be used simultaneously */\ncontent: open-quote chapter_counter;\n\n/* Global values */\ncontent: inherit;\ncontent: initial;\ncontent: unset;\n</counter></uri></string></code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/content
```

Comment:
Took me quite a while to find the cause, turns out that [Mojo::DOM](https://github.com/duckduckgo/zeroclickinfo-fathead/pull/426/commits/0aa462e3da84b38c86c53b47a78efe306e0b15f4) was the one decoding the html entities.